src/main/java/com/example/demo/entity/Team.java

```diff
@@ -0,0 +1,24 @@
+package com.example.demo.entity;
+
+import lombok.Data;
+
+@Data
+//@Setter
+//@Getter
+
+public class Team {
+    private String teamName;
+    private String captain;
+    private String coach;
+    private Integer points = (int) (Math.random() * 10);
```

Comment:
I would have used this `java.util.Random#nextInt(int)` in order to avoid class casting.